spec/mailers/notify_spec.rb

```diff
@@ -81,7 +86,10 @@
 
   describe 'for users that signed up, the email' do
     let(:example_site_path) { root_path }
-    let(:new_user) { create(:user, email: 'newguy@example.com', password: "securePassword") }
+
+    let(:new_user) do
+      create(:user, email: 'newguy@example.com', password: "securePassword")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.